src/composables/useAxios.js

```diff
@@ -16,7 +16,7 @@ const useAxios = () => {
       return Promise.reject(error)
     })
 
-    if (!DEBUG_MODE || DEBUG_MODE === 'false') {
+    if (!DEBUG_MODE) {
```

Comment:
nuxt uses destr to parse env variables so `NUXT_PUBLIC_DEBUG_MODE=false` will be literally `false`

Reply:
yeah, right. Make sense